### PR TITLE
IGVF-328-Expand-HumanDonor-ethnicity

### PR DESCRIPTION
**Pull request by corismall**

### CODE REVIEW

File: src/igvfd/schemas/changelogs/human_donor.md [hunk 1]
```diff
@@ -1,5 +1,13 @@
 ## Changelog for *`human_donor.json`*
 
+### Minor changes since schema version 3
+
+* Add `African Caribbean` to `ethnicity` enum.
```
Comment (gabdank) [inline src/igvfd/schemas/changelogs/human_donor.md:5]: please make it a single line listing all the additions to ethnicities

Comment (gabdank) [reply src/igvfd/schemas/changelogs/human_donor.md:5]: * Add `ethn1`,`etn2`, `etn3`,... to `ethnicities`

Comment (gabdank) [reply src/igvfd/schemas/changelogs/human_donor.md:5]: no need to say it is an enum

**Review by gabdank — CHANGES_REQUESTED** [review]
add human donor inserts utilizing your introduced ethnicities

**Comment by corismall** [comment]
accidentally clicked on merge branch instead of rebase

**Review by gabdank — APPROVED** [review]

**Review by gabdank — APPROVED** [review]

**Pull request merged**